test cases/common/134 generated llvm ir/main.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+
+unsigned square_unsigned (unsigned a);
+
+int
+main (int argc, char * argv[])
+{
+  unsigned int ret = square_unsigned (2);
+  if (ret != 4) {
+    printf("Got %u instead of 4\n", ret);
```

Comment:
Doesn't really matter in tests, and this is actually copied from `tests/common/126*/main.c`.